src/components/ReduxAppWrapper/store/reducers/captures.ts

```diff
@@ -0,0 +1,50 @@
+import * as constants from '../../constants'
+import { cleanFalsy } from '~utils/array'
+import { omit } from '~utils/object'
+import type { CaptureActions, CaptureState } from '../../types'
+
+type StateKeys = keyof CaptureState
+
+const initialState: CaptureState = {}
+
+const stateKey = (arr: string[]) => cleanFalsy(arr).join('_')
+const getKeyByCaptureId = (captures: CaptureState = {}, captureId: string) =>
+  (Object.keys(captures) as StateKeys[]).find(
+    (key) => captures[key].id === captureId
+  )
+
+export default function captures(
+  state = initialState,
+  action: CaptureActions
+): CaptureState {
+  if (action.type === constants.RESET_STORE) {
+    return initialState
+  }
+
+  if (action.type === constants.CAPTURE_DELETE) {
+    const key = stateKey([action.payload.method, action.payload.side])
```

Comment:
This implementation doesn't match with the `actions.deleteCapture()` calls [here](https://github.com/onfido/onfido-sdk-ui/blob/3f19bca504b28be97bf350d2597beff1a726ea88/src/components/Capture/Document.js#L55) and [here](https://github.com/onfido/onfido-sdk-ui/blob/3f19bca504b28be97bf350d2597beff1a726ea88/src/components/Capture/Face.js#L58), because those calls don't come with any payloads.

Reply:
Looks like a bug to me!